src/components/network/geo-data.js

```diff
@@ -15,6 +15,29 @@ const linePositionIndexer = (map, line) => {
     return map;
 };
 
+function distance(lat1, lon1, lat2, lon2, unit) {
```

Comment:
Do you had a look at existing library for that? Maybe deckgl/mapbox alreadu have one.

Reply:
Done